tests/hypothesis/to_unit_test.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+# @author Jan-Lukas Wynen
+import pytest
+from hypothesis import given
+from hypothesis import strategies as st
+
+import scipp as sc
+
+
+@given(st.floats(min_value=1e-300, max_value=1e300))
+def test_to_unit_value_1(x):
```

Comment:
Why 'value_1' ?

Reply:
Ah, but I forgot to remove the digits...